### ISSUE TITLE
[accessibility] rarity fish catch sounds aren't subtitled

### ISSUE THREAD

**Issue by Tralomine** [kleiwright/matcha-flavoured#147]
there's probably a lot more custom sounds that don't have correctly associated subtitles, but that's the one I noticed, it's an accessibility issue as deaf people cannot know what rarity fish they caught (and other useful sound cues)
